implementations/build-squeak.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+if [ "$1" = "style" ]
+then
+  exit 0
+fi
+
+set -e # make script fail on first error
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source $SCRIPT_PATH/script.inc
+
+if [ ! -e "$SCRIPT_PATH/squeak.sh" ]; then
+  INFO Get Squeak VM
+  pushd $SCRIPT_PATH
+  get_web_getter
+  $GET http://ftp.squeak.org/5.1/Squeak5.1-16549-32bit/Squeak5.1-16549-32bit-All-in-One.zip
+  unzip -q Squeak5.1-16549-32bit-All-in-One.zip
+  cp Squeak5.1*.app/Contents/Resources/Squeak5.1*.image ../benchmarks/Smalltalk/Squeak.image
+  cp Squeak5.1*.app/Contents/Resources/Squeak5.1*.changes ../benchmarks/Smalltalk/Squeak.changes
+  cp Squeak5.1*.app/Contents/Resources/Squeak*.sources ../benchmarks/Smalltalk/
+
+  # This can be dropped once https://github.com/travis-ci/travis-build/pull/879 is merged and in production
+  if [[ "$(uname -s)" = "Linux" ]]; then
+      wget -q https://raw.githubusercontent.com/hpi-swa/smalltalkCI/master/utils/set_rtprio_limit.c
+      gcc -o set_rtprio_limit set_rtprio_limit.c
+      chmod +x ./set_rtprio_limit
+      sudo ./set_rtprio_limit $$
```

Comment:
The high priority heartbeat thread is used to better capture input events AFAIK. So I'd say you could just use an itimer VM instead and remove that piece of code. Shouldn't affect the effectiveness of the JIT at all. Maybe even use [one of these](https://github.com/OpenSmalltalk/opensmalltalk-vm/releases/tag/201701281910).